contrib/ansible/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "ansible"
+pkgver = "10.3.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+checkdepends = [
+    "ansible-core",
+    "python-pytest",
+    "python-pytest-mock",
+    "python-pytest-xdist",
+]
+depends = ["ansible-core"]
+pkgdesc = "Radically simple IT automation"
+maintainer = "Mara <177581589+catgirlconspiracy@users.noreply.github.com>"
+license = "GPL-3.0-or-later"
+url = "https://ansible.com"
+source = f"$(PYPI_SITE)/a/ansible/ansible-{pkgver}.tar.gz"
+sha256 = "6144fb4bc785f917f86b1b0b6eadc9b894e9751ff9e9a7875afcfa2f74581ffd"
+# many collections either require additional Python modules, or the tests
+# require dependencies that aren't in the tarball, or they're just broken
+# (possibly due to Python version compat issues)
```

Comment:
you could run only the ones that work

Reply:
I tried excluding all the tests that didn't work

I gave up at ... 15 or 20 or so I think because it seemed like a lot of work and didn't feel worth it

like, I can set it up if you think it makes sense, but it's really quite a lot that don't test

Reply:
sure, i guess !check is fine then